vitrina/datasets/search_indexes.py

```diff
@@ -12,7 +12,8 @@ class DatasetIndex(SearchIndex, Indexable):
     text = CharField(document=True, use_template=True)
     lt_title = CharField(model_attr='lt_title')
     en_title = CharField(model_attr='en_title')
-    organization = IntegerField(model_attr='organization__pk', faceted=True)
+    management_area = MultiValueField(model_attr='management_area', faceted=True, null=True)
```

Comment:
```suggestion
    jurisdiction = MultiValueField(model_attr='management_area', faceted=True, null=True)
```

Be consistent with `Organization.jurisdiction` naming.